samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml

```diff
@@ -69,40 +33,17 @@ spec:
           mountPath: /cache
         - name: layers-dir
           mountPath: /layers
-    - name: step-build
-      image: docker.io/paketobuildpacks/builder:full
-      securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
-      command:
-        - /cnb/lifecycle/builder
-      args:
-        - -app=/workspace/source/$(build.source.contextDir)
-        - -layers=/layers
-        - -group=/layers/group.toml
-        - -plan=/layers/plan.toml
-      resources:
-        limits:
-          cpu: 500m
-          memory: 1Gi
-        requests:
-          cpu: 250m
-          memory: 65Mi
-      volumeMounts:
-        - name: layers-dir
-          mountPath: /layers
-    - name: step-export
+    - name: step-buildpacks
```

Comment:
`step-buildpacks` -> `step-create`? mainly because the binary is call `creator` and we know it creates the image

Reply:
I also asked myself what a good name would be. I used buildpacks because I think we should try to use identifying names within our sample build strategies. The reason is that the step name will at the end show up in an error message when the step failed. I have the [troubleshooting](https://cloud.ibm.com/docs/codeengine?topic=codeengine-troubleshoot-build) in mind where based on the step name, we give the user tips on what could be the root cause. If we use step names that are (eventually) used in different build strategies, then such documentation will become slightly more complicated (we'll have to distinguish based on strategy then).

If we do not consider this important, then your name is fine. We could also take the same as in kaniko (build-and-push).

Reply:
I would keep here `step-create` and then modify this in our side. Here `step-create` makes more sense.

Reply:
Step name changed.

Reply:
buildpack task in Tekton catalog uses `step-create`: https://github.com/tektoncd/catalog/blob/master/task/buildpacks/0.2/buildpacks.yaml#L85, FYI.